### DISCUSSION TITLE
Is it not possible to add new websites until the current block ends after update?

### DISCUSSION THREAD

**Discussion by CartoonJ** [SelfControlApp/selfcontrol#726] [Q&A] [answered]
<img width="434" alt="Screen Shot " src="https://user-images.githubusercontent.com/89230992/136303723-0d13b22d-c972-4d85-8584-1ec581de2a3f.png">

A bit of a bummer because I still have ~20days to go until this block section  ends.

**Comment by cstigler** [comment, marked as answer]
Yep, that's the deal! If that's something you need to do, just downgrade to 3.0.3 (still available on [our website](https://selfcontrolapp.com/)) for now, and pretend the 4.0 release comes out in 20 days :)

**Reply by CartoonJ** [reply]
It works! Thanks for replying and creating this incredible app!
